env.mjs

```diff
@@ -19,7 +19,11 @@ export const env = createEnv({
    * isn't built with invalid env vars. To expose them to the client, prefix them with
    * `NEXT_PUBLIC_`.
    */
-  client: {},
+  client: {
+    NEXT_PUBLIC_ROOT_ORG_ID: z
+      .string()
+      .default("3121a8a0-c548-4d14-a313-630c3b739858"),
```

Comment:
should we expose this?

Reply:
yea there's nothing wrong with exposing this imo. it gets used either way in production for certain whoamis